src/insert.ts

```diff
@@ -11,18 +11,12 @@ logger.debug(
   `\nRunning Service:\n '${SERVICE}'\n mode: ${NODE_ENV}\n stage: '${AWS_STAGE}'\n region: '${AWS_REGION}'\n\n`,
 );
 
-const handler = async (event: SQSEvent, _context: Context, callback: Callback) => {
-  try {
-    logger.debug(`Function triggered with '${JSON.stringify(event)}'.`);
-    await eventHandler(event);
+const handler = async (event: SQSEvent): Promise<SQSBatchResponse> => {
+  logger.debug(`Function triggered with '${JSON.stringify(event)}'.`);
+  const batchItemFailures = await eventHandler(event);
 
-    logger.info('Data processed successfully.');
-    callback(null, 'Data processed successfully.');
-  } catch (error) {
-    logger.info('Data processed unsuccessfully.');
-    logger.error('', error);
-    callback(new Error('Data processed unsuccessfully.'));
-  }
+  logger.info(`Data processed successfully with ${batchItemFailures.batchItemFailures.length} failures`);
```

Comment:
I don't think that we need to have this log

Reply:
I'm happy with this either way

Reply:
I'm not fussed, but either way I think both handlers should be consistent. So either they both have it, or neither have it.